Emby.Server.Implementations/ApplicationHost.cs

```diff
@@ -538,8 +539,21 @@ private IEnumerable<Task> StartEntryPoints(IEnumerable<IServerEntryPoint> entryP
 
                 Logger.LogDebug("Starting entry point {Type}", entryPoint.GetType());
 
-                yield return entryPoint.RunAsync();
+                try
+                {
+                    entryPointTasks.Add(entryPoint.RunAsync());
+                }
+                catch (MissingMethodException ex)
+                {
+                    Logger.LogError(ex, "Failed to start entry point for {Type}, there is a missing method", entryPoint.GetType());
+                }
+                catch (Exception ex)
```

Comment:
As I mentioned, I'm not sure this does anything since the task isn't being awaited in which case I don't think we can catch anything.

Task.WhenAll typically throws an aggregateexception